tox.ini

```diff
@@ -1,28 +1,32 @@
 [tox]
-envlist = py3{5,6}-{debug,release}
+envlist = py3{5,6,7}-{debug,release}
 skip_missing_interpreters = True
-skipsdist = True
 
 [testenv]
 deps =
     -r{toxinidir}/requirements.txt
+commands = pytest tests {posargs}
 
-basepython:
-    py35: python3.5
-    py36: python3.6
+[testenv:flake8]
+skipsdist = True
+skip_install = True
+deps =
+    -r{toxinidir}/requirements.txt
+commands = flake8 --show-source aioelasticsearch tests setup.py
+
+[testenv:isort]
+skipsdist = True
+skip_install = True
+deps =
+    -r{toxinidir}/requirements.txt
```

Comment:
I believe this env doesn't require the whole load of deps. just `isort` should be enough.

Reply:
No, just isort is not enough, it needs to install dependencies in order to sort imports correctly.